src/group_theory/perm/cycle_type.lean

```diff
@@ -241,4 +242,69 @@ def partition (σ : perm α) : partition (fintype.card α) :=
 
 end partition
 
+/-!
+### 3-cycles
+-/
+
+section is_three_cycle
+
+variables [decidable_eq α] {σ : perm α}
```

Comment:
Maybe inline these assumption for the definition, and then open a namespace?
Many names are now prefixed by `is_three_cycle`.
As of recently, you can write `lemma _root_.some_namespace.foobar` when you are inside a `namespace` to declare an occasional lemma outside of that namespace.